docs/workWithSheet.md

```diff
@@ -0,0 +1,573 @@
+# Working with Sheet Operations
+
+[ [Overview](#overview) ]  [ [Operation details](#operation-details) ]  [ [Sample configuration](#sample-configuration) ]
+
+
+## Overview
+
+The following operations allow you to carry out sheet operations. Click an operation name to see details on how to use it.
+
+
+
+| Operation | Description |
+| ------------- |-------------|
+| [addSheetBatchRequest](#Add-Sheet-Using-BatchRequest) | Add a sheet or multiple sheets to a spreadsheet |
+| [deleteSheetBatchRequest](#Delete-Sheet-Using-BatchRequest) | Remove a sheet or multiple sheets from a given spreadsheet |
+| [getSheetMetaData](#Retrieve-Sheet-metadata) | Provides the sheet metadata within a given spreadsheet |
+| [updateSheetPropertiesBatchRequest](#Update-Sheet-Properties-Using-BatchRequest) | Update all sheet properties |
+| [copyTo](#Copy-Sheet-Data) | Copy a single sheet from a spreadsheet to another spreadsheet |
+
+
+## Operation details
+
+This section provides details on each of the operations.
+
+
+
+#### Add Sheet Using BatchRequest
+
+This method allow you to add new sheets to an existing spreadsheet. You can specify the sheet properties for the new sheet. It is an error to provide a title that is used for an existing sheet.
+
+
+
+###### addSheetBatchRequest
+
+```xml
+<googlespreadsheet.addSheetBatchRequest>
+     <spreadsheetId>{$ctx:spreadsheetId}</spreadsheetId>
+     <requests>{$ctx:requests}</requests>
+     <fields>{$ctx:fields}</fields>
+</googlespreadsheet.addSheetBatchRequest>
+```
+
+
+
+###### Properties
+
+* spreadsheetId :- Unique value of the spreadsheet
+* requests :- It contains data that is a kind of update to apply to a spreadsheet. To add multiple sheets within the spread sheet, need to repeat `addSheetBatchRequest` property within the `requests` attribute as below in the sample request.
+* fields [Optional] :- Specifying which fields to include in a partial response. For the following request only the `spreadsheetId` will be included in the response.
+
+
+
+###### Sample request
+
+Following is a sample request that can be used for addSheetBatchRequest operation. With the following request we can specify multiple sheet properties such as sheet name("Expenses1", "Expenses2"), sheet type("GRID") and dimension ((50,10), (70,10)) of the sheet as an array. Here we specify fields property to get partial response. Here, `spreadsheetId`, `replies` will be included in the response. `replies` contains properties such as sheet name, type, row, column count, sheetId.
```

Comment:
The sample request given below can be used as the addSheetBatchRequest operation. The request specifies the multiple sheet properties, such as the sheet name ("Expenses1", "Expenses2"), sheet type ("GRID"), and the dimension ((50,10), (70,10)) of the sheet as an array. The fields property is specified to get a partial response. The `spreadsheetId`, `replies` values will be included in the response. `replies` contains properties such as sheet name, type, row, column count, and sheetId.